extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -519,4 +519,13 @@ private boolean isEqual(State state1, State state2) {
 
         return configStatusMessages;
     }
+
+    @Override
+    public String getDeviceId() {
+        try {
+            return bridge.getFullConfig().getConfig().getMACAddress().replaceAll(":", "").toLowerCase();
```

Comment:
I cannot use 
```` 
getThing().getConfiguration().get(SERIAL_NUMBER)
```` 
because when manual configuring, the serial number does not reflect the MAC address of the bridge, this could be any value. I have to get it from the device directly. But I cannot do it like the HueBridgeDiscoveryParticipant, as it relies on the UPnP result what we do not have here. Therefore I need to use my fourth way.